README.md

```diff
@@ -281,7 +281,7 @@ For authentication you can also use Travis CI secure environment variable:
 * **password**: PyPI Password.
 * **server**: Optional. Only required if you want to release to a different index. Follows the form of 'https://mypackageindex.com/index'. Defaults to 'https://pypi.python.org/pypi'.
 * **distributions**: Optional. A space-separated list of distributions to be uploaded to PyPI. Defaults to 'sdist'.
-* **skip_upload_docs**: Optional. When set to `true`, documentation is not uploaded. Defaults to `false`.
+* **skip_upload_docs**: Optional. When set to `false`, documentation is uploaded. Defaults to `true`.
```

Comment:
> Ah, right. Maybe we should remove this option altogether?

If removing it won't break builds which currently set it, I think this is the way to go. Perhaps just leave t in the docs but mark it as "deprecated and no longer having any effect".

> Or, is there an PyPI-compatible server that accepts the docs upload?

readthedocs.com is now the recommended service for that, but it works differently by pulling directly from the repository.